src/axom/core/utilities/nvtx/interface.cpp

```diff
@@ -31,34 +28,22 @@ static struct settings_t
 // NVTX interface implementation
 //------------------------------------------------------------------------------
 
-void set_color( Color c )
-{
-  Settings.color = static_cast< uint32_t >( c );
-}
+void set_color(Color c) { Settings.color = static_cast<uint32_t>(c); }
 
 //------------------------------------------------------------------------------
-Color get_color( )
-{
-  return static_cast< Color >( Settings.color );
-}
+Color get_color() { return static_cast<Color>(Settings.color); }
 
 //------------------------------------------------------------------------------
-void set_category( uint32_t category )
-{
-  Settings.category = category;
-}
+void set_category(uint32_t category) { Settings.category = category; }
 
 //------------------------------------------------------------------------------
-uint32_t get_category( )
-{
-  return Settings.category;
-}
+uint32_t get_category() { return Settings.category; }
```

Comment:
There is a setting regarding single-line functions.  
Axom was not consistent about this.  I allowed this in our config file.